spec/features/idv/in_person_threatmetrix_spec.rb

```diff
@@ -0,0 +1,310 @@
+        UspsInPersonProofing::Mock::Fixtures.request_enroll_response,
+      )['enrollmentCode']
+      expect(page).to have_content(t('in_person_proofing.headings.barcode').tr(' ', ' '))
+      expect(page).to have_content(Idv::InPerson::EnrollmentCodeFormatter.format(enrollment_code))
+      expect(page).to have_content(
+        t('in_person_proofing.body.barcode.deadline', deadline: deadline),
+      )
+      expect(page).to have_content('MILWAUKEE')
+      expect(page).to have_content('Sunday: Closed')
+
+      # signing in again before completing in-person proofing at a post office
+      Capybara.reset_session!
+      sign_in_live_with_2fa(user)
+      visit_idp_from_sp_with_ial2(:oidc)
+      expect(page).to have_current_path(idv_in_person_ready_to_verify_path)
+    end
+
+    it 'handles profiles in fraud review after 30 days', allow_browser_log: true do
+      sign_in_and_2fa_user(user)
+      begin_in_person_proofing(user)
+      complete_prepare_step(user)
+      complete_location_step
+      complete_state_id_step(user)
+
+      # ssn page
+      complete_ssn_step(user, tmx_status)
+      complete_verify_step(user)
+      complete_phone_step(user)
+      complete_enter_password_step(user)
+      acknowledge_and_confirm_personal_key
+
+      profile = InPersonEnrollment.last.profile
+      profile.deactivate_for_fraud_review
+      expect(profile.fraud_review_pending_at).to be_truthy
+      expect(profile.fraud_rejection_at).to eq(nil)
+      profile.update(fraud_review_pending_at: 31.days.ago)
+      FraudRejectionDailyJob.new.perform(Time.zone.now)
+
+      # profile is rejected
+      profile.reload
+      expect(profile.fraud_review_pending_at).to be(nil)
+      expect(profile.fraud_rejection_at).to be_truthy
+    end
+
+    context 'Fraud decisioning is completed' do
+      before do
+        complete_entire_ipp_flow(user, tmx_status)
+      end
+
+      context 'User passes IPP and passes TMX Review' do
+        it_behaves_like 'initially shows the user the barcode page'
+
+        it_behaves_like 'shows the user the Please Call screen'
+
+        it 'does not allow the user to restart the IPP flow', allow_browser_log: true do
+          deactivate_profile_update_enrollment(status: :passed)
+
+          visit_idp_from_sp_with_ial2(sp)
+          expect(page).to have_current_path(idv_please_call_path)
+          page.visit('/verify/welcome')
+          expect(page).to have_current_path(idv_please_call_path)
+          page.visit('/verify/in_person/document_capture')
+          expect(page).to have_current_path(idv_please_call_path)
+        end
+
+        it 'shows the user the successful verification screen after passing TMX review',
+           allow_browser_log: true do
+          deactivate_profile_update_enrollment(status: :passed)
+          visit_idp_from_sp_with_ial2(sp)
+          expect(page).to have_current_path(idv_please_call_path)
+
+          review_pass.run(args: [user.uuid], config:)
```

Comment:
@svalexander One thing I forgot to add here is end.to(change { ActionMailer::Base.deliveries.count }.by(1)), as done [here](https://github.com/18F/identity-idp/blob/cecba48397c977328c88ba173d9e60afa1ae1f96/spec/controllers/idv/enter_password_controller_spec.rb#L841). The idea is to see if we can check verify that the notification email is sent. Next comment is identical to this one.